licenses/licenses.xml

```diff
@@ -0,0 +1,378 @@
+<?xml version='1.0'?>
```

Comment:
Currently this (and the html equivalent) only contains info on the Node.js/npm dependencies. In our container image layer, we also install RPMs for `dejavu-sans-fonts`, and `phantomjs`. We should have license info recorded on those also. It'll be a lot easier if that's a separate bunch of xml/html files, but even then, I'm not sure how best to keep those up-to-date. I'll see what I can do!

Reply:
Perfectly fine to keep the files separate for Node and RPM licenses. RPMs change very rarely, so updating them would be not an usual task (and could be manual), while the Node files would be updated much more frequently (and with automation).

Reply:
We can also have multiple license.xml files within the root directory. The structure is:
./licenses/PRODUCT/*

This was designed around the potential scenario where we ship a product made up of products, so you could have RHMAP, RHOAR, EAP all in the one product and they each have their own folder for all the local file info + the HTML + the XML file.

So we could get creative there and differentiate between RHMAP licenses and RPM licenses (if that's a requirement). The naming requirements aren't hard enforced so you could do RHMAP and RHMAP_MISC for example -- we have no formal request that the names have to be a certain way.